go/adbc/driver/flightsql/flightsql_adbc_test.go

```diff
@@ -838,6 +875,34 @@ func (ts *TimeoutTestSuite) TestGetFlightInfoTimeout() {
 	ts.NotEqual(adbc.StatusNotImplemented, adbcErr.Code)
 }
 
+func (ts *TimeoutTestSuite) TestDontTimeout() {
```

Comment:
Can we also add a test that should timeout? e.g. set the timeout to 0.01 seconds and ensure it actually fails.

Reply:
all the previous tests do that. this is one of the only tests in this group that doesn't timeout